### ISSUE TITLE
AVT 1 - React Search Component, DAP violation when X button is present 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3454]
## Environment
macOS 10.14.5
Chrome 75.0.3770.100
VoiceOver
DAP 18 w/ July 2019 ruleset

## Description
1. Tab to Search field
2. Enter search text so the X button displays
3. Run DAP 
See attached screenshot for violations. 
Note: Fixing this issue should fix issue #3334
![Screen Shot 2019-07-18 at 2 47 02 PM](https://user-images.githubusercontent.com/21676914/61487449-8afde280-a96b-11e9-8791-9c5237529d29.png)

**Comment by jendowns** [comment]
Hey @dakahn I believe I have a fix for this here: https://github.com/carbon-design-system/carbon/pull/3483

EDIT: nevermind, this is the PR -- https://github.com/carbon-design-system/carbon/pull/3365
